samples/drivers/lora/send/README.rst

```diff
@@ -0,0 +1,36 @@
+.. zephyr:code-sample:: lora
+   :name: lora send
+   :relevant-api: lora_radio
+
+   Transmit a preconfigured payload every second using the lora radio.
+
+Overview
+********
+
+This sample demonstrates how to use the lora radio driver to configure
+the encoding settings and send data over the radio.
+
+Transmitted messages can be received by building and flashing the accompanying
+lora receive sample on another board.
+
+Building and Running
+********************
+
+Build and flash the sample as follows, changing ``b_l072z_lrwan1`` for
+your board, where you board has a default lora radio.
+
+.. zephyr-app-commands::
+   :zephyr-app: zephyr/samples/drivers/lora/send
+   :host-os: unix
+   :board: b_l072z_lrwan1
+   :goals: build flash
+   :compact:
+
+Sample Output
+=============
+
+.. code-block:: console
+    [00:00:00.531,000] <inf> lora_send: Data sent!
```

Comment:
I guess there needs to be a newline betwee `.. code-block:: console` and first log line. It looks like this is the reason why documentation build failed.